environment.conf.json

```diff
@@ -14,6 +22,8 @@
     "siteId": 14
   },
   "language": {
+    "title": "Signalen Informatievoorziening Amsterdam",
+    "shortTitle": "SIA",
```

Comment:
Shouldn't this prop be a child of the `meta`/`manifest` prop?

Reply:
I've considered it, but thought it would be more convenient to have all language together.